neofs-local/HOWTO.md

```diff
@@ -41,84 +45,77 @@ ERROR: error pulling image configuration: Get https://registry-1.docker.io/v2/li
 ```
 
 ## Neo Private Network
-*If you want to use NeoFS with NeoPrivateNetwork, [**read this note**](./Blockchain.md)*
+*If you want to use NeoFS with NeoPrivateNetwork, [**read this note**](./Blockchain.md).*
+
+**Attention!** To start NeoFS with blockchain indexer uncomment these lines in `./neofs-local/.env`:
 
-**Attention** after this steps uncomment these lines in `./neofs-local/.env`:
 ```
-# # Indexer settings
-# NEOFS_WORKERS_INDEXER_DISABLED=false
-# NEOFS_INDEXER_START_FROM=4000
-# NEOFS_INDEXER_ENDPOINT=http://privnet:30333
-# NeoFS Smart Contract scripthash
+## Indexer settings
+#NEOFS_WORKERS_INDEXER_DISABLED=false
+#NEOFS_INDEXER_START_FROM=4000
+#NEOFS_INDEXER_ENDPOINT=http://privnet:30333
+## NeoFS Smart Contract scripthash
 #NEOFS_INDEXER_ADDRESSES_0=dcc5902c9e8c63286894015ffd27097fd0ac9656
 ```
-And change `NEOFS_INDEXER_ADDRESSES_0` to SC scripthash used in NeoPrivateNetwork,
+
+**Note:** don't forget to change `NEOFS_INDEXER_ADDRESSES_0` variable to 
+NeoFS smart contract scripthash used in NeoPrivateNetwork,
  e.g.`dcc5902c9e8c63286894015ffd27097fd0ac9656`.
  
-There are operations that require user's deposit e.g. container creation.
-If you want to start NeoPrivateNetwork, then you might want to do operations with actual payment:
-create user's deposit, pay for the container creation, pay for the storage etc. 
-To do that **comment** these lines in `./neofs-local/.env`:
-
+There are operations that require user's deposit.
+Comment these lines in `./neofs-local/.env` to **activate** storage payment and 
+container creation payment.
 ```
 # Disable pay of the creation containers
 NEOFS_BOOKKEEPER_CONTAINER_NODE_COUNT_FACTOR=0
 NEOFS_BOOKKEEPER_CONTAINER_CONTAINER_CAPACITY_FACTOR=0
-
-# Do not delete unpaid objects
```

Comment:
that comment not relevant anymore, right?

Reply:
Yes. Actually `.env` file never had this comment.